modules/cluster-logging-visualizer-indices.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-visualizer.adoc
+
+[id="cluster-logging-visualizer-indices_{context}"]
+= Defining Kibana index patterns
+
+An index pattern defines the Elasticsearch indices that you want to visualize. To explore and visualize data in Kibana, you must create an index pattern. 
+
+.Prerequisites
+
+* A user must have the `cluster-admin` role, the `cluster-reader` role, or both roles in order to list the *infra* and *audit* indices in Kibana. 
```

Comment:
I think we shouldn't mention the audit indices here. the audit logs are not collected when using the default log store.

Below is the explanation from Chris about why no audit indices in Kibana.
Exactly. 
Now, there is no "forcing" factor for that and customers can choose to forward audit logs explicitly to our ES by using the "default" output type and this is when you would see audit logs in that index. But, again - not the logging team nor the security team would recommend it.